commit f2cc3776a219f025845944d04fe0a3124f58068a
Author: Rafal Krypa <r.krypa@samsung.com>
Date:   Wed Feb 3 14:46:49 2010 +0900

    src/Makefile: workaround for Scratchbox, always use -lcrypt for sulogin.

diff --git a/src/Makefile b/src/Makefile
index c466bf8..4a1fec1 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -64,9 +64,10 @@ endif
 
 
 # Additional libs for GNU libc.
-ifneq ($(wildcard /usr/lib/libcrypt.a),)
+## Workaround for Scratchbox
+#ifneq ($(wildcard /usr/lib/libcrypt.a),)
 LCRYPT		= -lcrypt
-endif
+#endif
 
 all:		$(BIN) $(SBIN) $(USRBIN)
 
